### PR TITLE
Feat: Added link of dynamic list of contributors

### LINKED ISSUE #61: Add Automatic Contributor List to README
There's GOT to be some way to show contributors automatically in the README. So that Github contributors can automatically be listed without needing to be added manually (would save me some time, and make sure people get some of the credit and thanks that they need).

### PR DESCRIPTION
**Pull request by asprazz**
Fixes #61 

Hi @engineerjoe440,

Please review and let me know if any changes are required. : )

Thanks

### CODE REVIEW

**Review by engineerjoe440 — APPROVED** [review]
Marvelous! Waiting for all the checks to pass, then we can merge it! Approving now, so you (or I) can merge as soon as CI wraps up.

**Comment by asprazz** [comment]
cropped

I was about to share one screenshot. Anyway thanks for merging.

**Pull request merged**
